curriculum/challenges/english/10-coding-interview-prep/rosetta-code/state-name-puzzle.md

```diff
@@ -15,7 +15,7 @@ dashedName: state-name-puzzle
 
 Write a function to solve the challenge for a given array of actual U.S. state names, and for fictional state names.
 
-The function should return an array. Each element should be an object in this form: `{"from":[],"to":[]}`. The `"from"` array should contain the original names and the `"to"` array should contain the resultant names.
+The function should return an array. Each element should be an object in this form: `{from:[],to:[]}`. The `"from"` array should contain the original names and the `"to"` array should contain the resultant names.
```

Comment:
make sure to adjust this line as requested

Reply:
I misinterpret it.
I thought that I should only remove quotation from Object.
I've done the changes.